packages/devtools_app/test/debugger_screen_variales_test.dart

```diff
@@ -0,0 +1,419 @@
+// Copyright 2020 The Chromium Authors. All rights reserved.
```

Comment:
typo in the file name
rename the file from:
`test/debugger_screen_variales_test.dart`
to
`test/debugger_screen_variables_test.dart`

Reply:
done